packages/drivers/odsp-driver/src/contracts.ts

```diff
@@ -25,7 +25,11 @@ export interface ISnapshotTreeEx {
  * Socket storage discovery api response
  */
 export interface ISocketStorageDiscovery {
-    // The id of the web socket
+    /**
+     * relaySessionId - identifies relay service session for this document.
+     * There is at most one session per document at any given time.
+     * Same as ITokenClaims.documentId, IConnect.id
+     */
     id: string;
```

Comment:
Should not we change from id to relaySessionId here?

Reply:
This, and other places like that describe today's wire protocol. While it's possible to change it (and it will be easier to do so here than in other places as there is no impact on FRS), it would need to be done on SPO side first. We can do it, but let's track it separately - this PR only does renames that are not affecting protocol (i.e. only client)